versioned_docs/version-2.8/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -156,35 +165,47 @@ spec:
 
-After installing a chart, you can find it in the **Installed Apps** tab. In this section you can upgrade or delete the installation, and see further details. When choosing to upgrade, the form and values presented will be the same as installation.
+To view all recent changes, click **Apps > Recent Operations** in the left navigation menu. From there you can view the calls, conditions, events, and logs.
 
-Most Rancher tools have additional pages located in the toolbar below the **Apps** section to help manage and use features. These pages include links to dashboards, forms to easily add Custom Resources, and additional information.
+After installing a chart, you can view it by clicking **Apps > Installed Apps** in the left navigation menu. You can upgrade or delete the installation, and see further details. Upgrading uses the same forms and values as you saw during inital installation. 
+
+Most Rancher tools have additional pages located in the toolbar below the **Apps** section to help manage and use the features. These pages include links to dashboards, forms to easily add Custom Resources, and additional information.
 
 :::caution
 
-If you are upgrading your chart using **Customize Helm options before upgrade**, please be aware that using the `--force` option may result in errors if your chart has immutable fields. This is because some objects in Kubernetes cannot be changed once they are created. To ensure you do not get this error you can:
+If you are upgrading your chart using **Customize Helm options before upgrade**, and your chart contains immutable fields, using the `--force` option may result in errors. This is because some objects in Kubernetes can't be changed after they're created. To prevent this error:
 
-  * Use the default upgrade option ( i.e do not use `--force` option).
+  * Use the default upgrade option (i.e don't use `--force`).
   * Uninstall the existing chart and install the upgraded chart.
-  * Delete the resources with immutable fields from the cluster before performing the `--force` upgrade.
+  * Delete the resources with immutable fields from the cluster before performing a forced upgrade.
 
 :::
 
-#### Legacy Apps
+### Legacy Apps
+
+The upgrade button isn't available for legacy apps on the **Apps > Installed Apps** page.
+
+If you want to upgrade an installed legacy app, the [legacy feature flag](../../advanced-user-guides/enable-experimental-features/enable-experimental-features.md) must be turned on. This flag is automatically turned on if you had a legacy app already running before you upgraded Rancher.
+
+1. Enable the [legacy feature flag](../../advanced-user-guides/enable-experimental-features/enable-experimental-features.md), if it isn't enabled already. This flag is enabled automatically if you upgrade Rancher while running a legacy app.
```

Comment:
```suggestion
1. Enable the [legacy feature flag](../../advanced-user-guides/enable-experimental-features/enable-experimental-features.md), if it isn't enabled already.
```
Since this is mentioned on line 196, is this repetitive, or do we want to mention it again in a different way here?